packages/tldr/tldr.0.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "coby@case.edu"
+authors: ["Jacob Rosales Chase <coby@case.edu>"]
+license: "MIT"
+homepage: "https://github.com/RosalesJ/tldr-ocaml/"
+bug-reports: "https://github.com/RosalesJ/tldr-ocaml/issues"
+dev-repo: "git+https://github.com/RosalesJ/tldr-ocaml.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "@install"]
+]
+
+depends: [
+  "dune" {build}
+  "core"
+  "lwt"
+  "ppx_jane"
+  "terminal_size"
+  "bos"
+  "cohttp"
+  "cohttp-lwt-unix"
+  "lwt_ssl"
+  "ssl" {= "0.5.5"}
```

Comment:
Is there no way of having a more relaxed constraint?

Reply:
Indeed ```lwt_ssl``` requires ```ssl```, but I require ```"ssl" {= "0.5.5"}``` because of an unfixed issue in [ssl 0.5.6](https://github.com/savonet/ocaml-ssl/issues/40).

I can, however, get rid of ```cohttp``` and ```lwt```

Reply:
You could say 

```
conflicts: [ "ssl" {= "0.5.6"} ]
```

so that you don't need to change the opam file to pick up the fix in a later version instead of having the ssl bound on the deps

Reply:
Is the patch in ssl.0.5.7?